packages/type/src/index.ts

```diff
@@ -1,11 +1,14 @@
-const { getPromiseDetails, getProxyDetails } =
-	typeof process === 'undefined'
-		? {
-				getPromiseDetails: () => undefined,
-				getProxyDetails: () => undefined
-		  }
-		: // @ts-ignore process.binding exists
-		  process.binding('util');
+// const { getPromiseDetails, getProxyDetails } =
+// 	typeof process === 'undefined'
+// 		? {
+// 				getPromiseDetails: () => undefined,
+// 				getProxyDetails: () => undefined
+// 		  }
+// 		: // @ts-ignore process.binding exists
+// 		  process.binding('util');
+
+import bindings from 'bindings';
```

Comment:
```suggestion
import bindings from 'bindings';
```

Lets remove commented out code yeah ;)